sandbox.c

```diff
@@ -135,7 +135,6 @@ int create_sandbox()
     ret = mount("tmpfs", "/", "tmpfs", MS_REMOUNT | MS_RDONLY, "size=0k");
     if (ret < 0) {
         fprintf(stderr, "cannot mount tmpfs on /tmp\n");
```

Comment:
it doesn't seem `/tmp`?

Reply:
yes the error message is wrong here it should mention "cannot remount / as read-only"

Reply:
I could add another commit to the PR that changes the error message, if you'd like

Reply:
typo is fixed in https://github.com/rootless-containers/slirp4netns/pull/170